### DISCUSSION TITLE
Lotus miner can not seal because of memory

### DISCUSSION THREAD

**Discussion by metalgear-dev** [filecoin-project/community#395] [Q&A]
Hello, everyone. I'm very interested in filecoin. And I want to implement it with lotus framework.
Now I'm using lotus with version 1.13.2. I use 128G RAM and swap with 256G.
But when I seal the sectors, error occurs like "sched: not scheduling on worker ea70b111-3c34-4af1-b27b-3a7fdca4f2db for withResources; not enough physical memory - need: 176808M, have 128847M"

There is a github closed issue like this one.
https://github.com/filecoin-project/lotus/issues/2109
And there, some one said that this issue had been already closed in the following url.
https://github.com/filecoin-project/lotus/pull/2155
FFI version was the problem(Update to 0.30.2). But I'm using FFI with version 0.30.4 
How can I deal with this error?
If anyone who can help me deal with this error, please let me know.

**Comment by gdragonHH** [comment]
硬件算力出海，专业技术团队 独立节点，欢迎大家加V了解：Hmcxxxxx
